VS2012/Adapter/ChutzpahMapperSettingsService.cs

```diff
@@ -22,6 +22,7 @@ public void MapSettings(ChutzpahUTESettings settings)
         {
             Settings.MaxDegreeOfParallelism = settings.MaxDegreeOfParallelism;
             Settings.EnabledTracing = settings.EnabledTracing;
+            Settings.BrowserArgs = settings.BrowserArguments;

```

Comment:
I am moving away from adding settings in the VS dialogs. Please add to the chutzpah.json file instead.